app/screens/AppNavigator/screens/Balances/screens/TokensVsUtxoScreen.tsx

```diff
@@ -0,0 +1,61 @@
+import { MaterialIcons } from '@expo/vector-icons'
+import React from 'react'
+import { View } from 'react-native'
+import { Text } from '../../../../../components'
+import { translate } from '../../../../../translations'
+import { tailwind } from '../../../../../tailwind'
+import { getTokenIcon } from '../../../../../components/icons/tokens/_index'
+
+export function TokensVsUtxoScreen (): JSX.Element {
+  return (
+    <View testID='token_vs_utxo_screen'>
```

Comment:
Use ScrollView, also can you make background white, hard to read with gray color. Since we're using the same format on CreateWalletinfo page @jtlntno-cake

Reply:
Will change to ScrollView and white background.

I noticed flex-1 doesn't work on the child items of ScrollView to achieve the following two scenarios:
- fill out the empty space when screen's height > content's height
- able to scroll through content when screen's height < content's height

Reply:
Flex-1 is for width right? Also, ScrollView will render a wrapper div, which is why you need to have contentContainerStyle. 

No need to take full screen especially on mobile, just allow the content to have the normal layout